keyboards/a_jazz/akc084/readme.md

```diff
@@ -0,0 +1,22 @@
+# A-JAZZ  AKC084
+
+* A customizable 84keys keyboard.
+![akc084](https://i.imgur.com/381vaD7.png)
+* Keyboard Maintainer: [Feng](https://github.com/fenggx-a-jazz)
+* Hardware Supported: [a-jazz](https://www.a-jazz.com)
+* Hardware Availability: [a-jazz](https://www.a-jazz.com)
```

Comment:
A couple things...
1. Can you please provide a link to this keyboard's store listing page? I'm unable to find anything regarding this keyboard on the internet.
2. Is this a wireless or "tri-mode" keyboard? I am able to find [Ajazz AKC087](https://ajazzstore.com/collections/all/products/ajazz-akc087), which is advertised as a "Bluetooth 5.0/2.4G Wireless & Type-C Wired" keyboard.

Reply:
https://ajazzstore.com/collections/all/products/ajazz-akp846?variant=43770515587125

Reply:
Thanks!